apps/explorer/src/components/OwnedCoins/components/CoinItem.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useFormatCoin } from '@mysten/core';
+import { type CoinStruct } from '@mysten/sui.js';
+
+import { ObjectLink } from '~/ui/InternalLink';
+import { Text } from '~/ui/Text';
+
+type CoinItemProps = {
+    coin: CoinStruct;
+};
+
+function CoinItem({ coin }: CoinItemProps): JSX.Element {
```

Comment:
No need to type the return on these (no need to make a PR for this, just for future reference)